backend/src/seeds/users/UserInterface.ts

```diff
@@ -9,4 +9,5 @@ export interface UserInterface {
   deletedAt?: Date
   publisherId?: number
   isAdmin?: boolean
+  password?: string
```

Comment:
Why? Do we want to have another password in seeded data?

Reply:
the password is used in the contribution factory to login the user